lua/lspconfig/server_configurations/eslint.lua

```diff
@@ -112,6 +112,11 @@ return {
         name = vim.fn.fnamemodify(new_root_dir, ':t'),
       }
 
+      -- Support flat config
+      if vim.fn.filereadable(new_root_dir .. '/eslint.config.js') == 1 then
```

Comment:
this file concat only work in *nix. so you may need support windows.

Reply:
I checked the code of `util.path.join` that's used in this file, and it just uses "/" to join the paths:
https://github.com/neovim/nvim-lspconfig/blob/32f0c0465ac3ee53a1a9c0f72602a7b9cecb31d3/lua/lspconfig/util.lua#L157-L159

The "/" should work on Windows as well, right?

Reply:
nope .windows separator is `\\` . the path will sanitize

```lua
local function sanitize(path)
    if is_windows then
      path = path:sub(1, 1):upper() .. path:sub(2)
      path = path:gsub('\\', '/')
    end
    return path
  end
```

Reply:
`local path_sep = vim.fn.has('win32') == 1 and '\\' or '/'`